.github/workflows/playwright.yml

```diff
@@ -0,0 +1,25 @@
+name: Tram-Deco Playwright Tests
```

Comment:
This file is almost entirely from the `npx playwright install`, and is probably the most bare-bones we could probably get this to be.